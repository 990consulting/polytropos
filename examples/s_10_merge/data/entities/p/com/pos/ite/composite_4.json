{
  "period_1": {
    "some_text": "P1(root)",
    "t_folder": {
      "some_text": "P1(folder)"
    }
  },
  "immutable": {
    "i_folder": {
      "some_text": null
    },
    "simple_keyed_list": {
      "nl_2": {"some_text": "222"}
    }
  }
}