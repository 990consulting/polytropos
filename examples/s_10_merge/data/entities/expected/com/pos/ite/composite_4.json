{
  "period_1": {
    "some_text": "P1(root)",
    "t_folder": {
      "some_text": "P1(folder)"
    }
  },
  "period_2": {
    "some_text": "P2(root)",
    "t_folder": {
      "some_text": "P2(folder)"
    },
    "t_list": [
      {"some_text": "P2(list)(A)"},
      {"some_text": "P2(list)(B)"}
    ]
  },
  "immutable": {
    "i_folder": {
      "some_text": null
    },
    "simple_keyed_list": {
      "nl_1": {"some_text": "111"},
      "nl_2": {"some_text": "222"}
    }
  }
}